koku/masu/processor/azure/azure_report_processor.py

```diff
@@ -167,10 +168,12 @@ def _create_cost_entry_product(self, row, report_db_accessor):
         """
         table_name = AzureCostEntryProductService
         instance_id = row.get('InstanceId')
+        additional_info = json.loads(row.get('AdditionalInfo'))
```

Comment:
We might want to consider using https://pypi.org/project/simplejson/ here since it's more performance oriented with the C extension.  I've never personally used it but would be happy to help investigate if you need.  Only reason for mentioning it here is that that we have seen fairly long line item processing times for AWS.

All this said... Azure has considerably fewer lines since it's already daily so we could also open an issue to switch to simplejson if necessary in the future.

Reply:
Implemented `ujson`